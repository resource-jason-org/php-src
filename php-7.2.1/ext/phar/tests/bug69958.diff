001+ Fatal error: Uncaught BadMethodCallException: phar "/home/php-src/php-7.2.1/ext/phar/tests/bug69958.tar" exists and must be unlinked prior to conversion in /home/php-src/php-7.2.1/ext/phar/tests/bug69958.php:3
001- Fatal error: Uncaught exception 'BadMethodCallException' with message 'phar "%s/bug69958.tar" exists and must be unlinked prior to conversion' in %s/bug69958.php:%d