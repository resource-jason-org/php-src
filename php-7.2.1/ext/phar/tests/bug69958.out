Fatal error: Uncaught BadMethodCallException: phar "/home/php-src/php-7.2.1/ext/phar/tests/bug69958.tar" exists and must be unlinked prior to conversion in /home/php-src/php-7.2.1/ext/phar/tests/bug69958.php:3
Stack trace:
#0 /home/php-src/php-7.2.1/ext/phar/tests/bug69958.php(3): PharData->convertToData(2)
#1 {main}
  thrown in /home/php-src/php-7.2.1/ext/phar/tests/bug69958.php on line 3